src/main/java/frc/robot/RobotContainer.java

```diff
@@ -88,4 +113,8 @@ private static double modifyAxis(double value) {
 
     return value;
   }
+
+  protected DrivebaseSubsystem getDrivebaseSubsystem() {
```

Comment:
Looks like this method is unused - what is it for?

Reply:
Ur right. Let's remove

Reply:
No use for this given the current structure anymore -- this was in anticipation of something that never ended up happening